Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ApplicationManagment/ApplicationNode.tsx

```diff
@@ -167,7 +167,7 @@ const ByApplicationNode: Application.Types.iByComponent = (props) => {
                     </div>
                 </div>
             </Modal>
-            <Warning Title={'Delete Application'} Message={'This will Delete this Application from the SSO System. This can have unintended consequences and cause the System to crash. Are you sure you want to continue?'}
+            <Warning Title={'Delete SSO Application - ' + editnewNode.Name} Message={'This will delete this Application from the SSO system. This can have unintended consequences and cause the system to crash. Are you sure you want to continue?'}
```

Comment:
We may have to make sure `editNewNode` can not be `null`? otherwise this may fail. There is a few pages where `null` is used to hide some things.